CD2/MyLA.md

```diff
@@ -0,0 +1,73 @@
+# MyLA queries for admin, communications, & reporting
+
+## Find courses with MyLA in the Navigation menu
+
+*This query is used to identify courses that have added MyLA to course navigation.*
+
+```
+SELECT co.value.enrollment_term_id
+  ,co.key.id
+  ,co.value.name
+  ,co.value.course_code
+  ,co.value.workflow_state
+  ,co.value.tab_configuration
+FROM udp-umich-prod.canvas.courses co
+WHERE co.value.enrollment_term_id = <TERM_ID>
+  AND co.value.workflow_state = 'available'
+  AND (co.value.tab_configuration like '%context_external_tool_28861"}%'
+      OR co.value.tab_configuration like '%context_external_tool_28861","hidden":null,"%')
+;
+```
+
+## Find teachers on courses with MyLA in the Navigation Menu
+
+*This query is used to find the names and uniqnames of teachers on MyLA courses for communications.*
+
+```
+SELECT DISTINCT ps.value.unique_id
+    ,us.value.name as teacher_name
+    ,en.value.type
+    ,ro.value.name
+    ,co.key.id as course_id
+FROM udp-umich-prod.canvas.enrollments en
+JOIN udp-umich-prod.canvas.pseudonyms ps on en.value.user_id = ps.value.user_id
+JOIN udp-umich-prod.canvas.users us on en.value.user_id = us.key.id
+JOIN udp-umich-prod.canvas.roles ro on en.value.role_id = ro.key.id
+JOIN udp-umich-prod.canvas.courses co on en.value.course_id = co.key.id
+WHERE co.key.id in  (
+    SELECT co.key.id
+    FROM udp-umich-prod.canvas.courses co
+    WHERE co.value.enrollment_term_id = 311
+        AND co.value.workflow_state = 'available'
+        AND (co.value.tab_configuration like '%context_external_tool_28861"}%'
+         OR co.value.tab_configuration like '%context_external_tool_28861","hidden":null,"%')
+    )
+AND en.value.type = 'TeacherEnrollment'
+AND en.value.workflow_state = 'active'
```

Comment:
since there are multiple teachers within a course, it is better to add a sort statement here to sort by co.key.id.

Reply:
another possibility is to join the `udp-umich-prod.mart_helper.context__course_offering__enrollment` data mart, where the instructure names and emails are already calculated as strings